doc/salobj_cscs.rst

```diff
@@ -234,29 +240,33 @@ Telemetry Loop Example
 
 Here is an example of how to write a telemetry loop.
 
-1. In the constructor (``__init__``): initialize::
+1. In the constructor (``__init__``): initialize:
+
+  .. code-block:: python
 
     self.telemetry_loop_task = salobj.make_done_future()
     self.telemetry_interval = 1  # seconds between telemetry output
 
-   Initializing ``telemetry_loop_task`` to an `asyncio.Future` that is already done makes it easier to test and cancel than initializing it to `None`.
+  Initializing ``telemetry_loop_task`` to an `asyncio.Future` that is already done makes it easier to test and cancel than initializing it to `None`.
+
+2. Define a ``telemetry_loop`` method, such as:
 
-2. Define a ``telemetry_loop`` method, such as::
+  .. code-block:: python
 
-    def telemetry_task(self):
+    async def telemetry_loop(self):
         while True:
-            # read and write telemetry
-            #...
+            #...read and write telemetry...
             await asyncio.sleep(self.telemetry_interval)
 
-3. Start and stop the telemetry loop in `BaseCsc.report_summary_state`::
+3. Start and stop the telemetry loop in `BaseCsc.report_summary_state`:
+
+  .. code-block:: python
 
     def report_summary_state(self):
         super().report_summary_state()
-        if self.summary_state in (salobj.State.DISABLED, salobj.State.ENSABLED):
+        if self.summary_state in (salobj.State.DISABLED, salobj.State.ENABLED):
             if self.telemetry_loop_task.done():
                 # telemetry loop is not running; start it
```

Comment:
I see your point. I personally think the comment is useful because it's not dead obvious that the task being done = the loop is not running, but I'll remove it.